allure-grpc/build.gradle.kts

```diff
@@ -0,0 +1,68 @@
+import com.google.protobuf.gradle.generateProtoTasks
+import com.google.protobuf.gradle.id
+import com.google.protobuf.gradle.ofSourceSet
+import com.google.protobuf.gradle.plugins
+import com.google.protobuf.gradle.protobuf
+import com.google.protobuf.gradle.protoc
+
+description = "Allure Grpc Integration"
+
+val agent: Configuration by configurations.creating
+
+plugins {
+    id("com.google.protobuf") version "0.8.16"
+}
+
+dependencies {
+    agent("org.aspectj:aspectjweaver")
+    api(project(":allure-attachments"))
+    implementation("io.grpc:grpc-api")
+    implementation("org.awaitility:awaitility")
```

Comment:
it's better to have no additional dependencies

Reply:
No way to drop grpc-api. The library provides an client interceptor interface and client calls, also much more stuff.

If it's about awaititlity, then the library provides a very convenient way to wait for events. I understand that this is not the best option, add a dependency due to 3 expectations in the code. 
I would be glad to get an advise how I can replace this with the current libraries in the project.

Reply:
Also added dependencies for pretty formatting